README.md

```diff
@@ -77,6 +77,12 @@ Once connected, the `open` event is fired on the `Connection` instance. If you'r
 
 **Note:** _If the local connection fails then try using 127.0.0.1 instead of localhost. Sometimes issues may arise when the local hostname has been changed._
 
+If `localhost` fails, use
+
+```js
+await mongoose.connect('mongodb://0.0.0.0:27017/my_database');
```

Comment:
I'm pleasantly surprised that connecting to `0.0.0.0` actually works. However, I don't want to rely on that or recommend that because [0.0.0.0 is "used to designate an invalid, unknown or non-applicable target"](https://en.wikipedia.org/wiki/0.0.0.0).